keyboards/ibm/model_m/yacobo/keyboard.json

```
@@ -0,0 +1,286 @@
+{
+    "manufacturer": "IBM",
+    "keyboard_name": "Model M (Yacobo)",
+    "maintainer": "sje-mse",
+    "bootloader": "stm32duino",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "extrakey": true,
+        "mousekey": true
+    },
+    "qmk": {
+        "locking": {
+            "enabled": true,
+            "resync": true
+        }
+    },
+    "indicators": {
+        "num_lock": "B11",
+        "caps_lock": "B10",
+        "scroll_lock": "B1",
+        "on_state": 0
+    },
+    "matrix_pins": {
+        "cols": ["B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "A10", "A9", "A8", "B15", "B14", "B13", "B12", "A0"],
+        "rows": ["A1", "A2", "A3", "A4", "A5", "A6", "A7","B0"],
+        "ghost": true
+    },
+    "processor": "STM32F103",
```

replace `bootloader` and `processer` with `development_board`
```suggestion
    "diode_direction": "COL2ROW",
    "features": {
        "bootmagic": true,
        "extrakey": true,
        "mousekey": true
    },
    "qmk": {
        "locking": {
            "enabled": true,
            "resync": true
        }
    },
    "development_board": "bluepill",
    "indicators": {
        "num_lock": "B11",
        "caps_lock": "B10",
        "scroll_lock": "B1",
        "on_state": 0
    },
    "matrix_pins": {
        "cols": ["B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "A10", "A9", "A8", "B15", "B14", "B13", "B12", "A0"],
        "rows": ["A1", "A2", "A3", "A4", "A5", "A6", "A7","B0"],
        "ghost": true
    },
```